### PR TITLE
Add install requires pyrsistent(ODM-3955)

### PR DESCRIPTION
**Pull request by AleksPark**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
     description='Genestack Python Client Library',
     long_description=__doc__,
     long_description_content_type="text/markdown",
-    install_requires=['keyring', 'requests', 'pyOpenSSL', 'jsonschema'],
+    install_requires=['keyring', 'requests', 'pyOpenSSL', 'jsonschema','pyrsistent'],
```
Comment (efimova-lena0) [inline setup.py:24]: I think need to add 
pyrsistent==0.16.0

Comment (vasiliy-bout) [reply setup.py:24]: and space is likely missing after comma

Comment (AleksPark) [reply setup.py:24]: Fixed

Comment (vasiliy-bout) [reply setup.py:24]: Sorry, but GitHub still shows the same code as before

Comment (vasiliy-bout) [inline setup.py:24]:
```suggestion
    install_requires=['keyring', 'requests', 'pyOpenSSL', 'jsonschema', 'pyrsistent==0.16.0'],
```

Comment (AleksPark) [reply setup.py]: I forgot to push it.
Now I've done

**Review by vasiliy-bout — APPROVED** [review]

**Pull request merged**